src/binding/python/Series.cpp

```diff
@@ -19,6 +19,7 @@
  * If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <pybind11/complex.h>
```

Comment:
Seems unrelated - let's check if we need to add it separately.

Reply:
Removed.